gwtbootstrap3/src/main/java/org/gwtbootstrap3/client/GwtBootstrap3EntryPoint.java

```diff
@@ -32,26 +34,27 @@
 public class GwtBootstrap3EntryPoint implements EntryPoint {
 
     /**
-     * Check to see if Boostrap is loaded already.
+     * Check to see if Bootstrap is loaded already.
      * 
-     * @return true is Boostreap loaded, false otherwise.
+     * @return true is Bootstrap loaded, false otherwise.
      */
-    private native boolean isBootstrapLoaded() /*-{
-        return typeof $wnd['jQuery'].fn.emulateTransitionEnd !== 'undefined'
-    }-*/;
+    @JsMethod
+    private static native boolean isBootstrapLoaded();
 
     /**
      * Check to see if jQuery is loaded already
      *
      * @return true is jQuery is loaded, false otherwise
      */
-    private native boolean isjQueryLoaded() /*-{
-        return (typeof $wnd['jQuery'] !== 'undefined');
-    }-*/;
+    @JsMethod
+    private static native boolean isjQueryLoaded();
 
     /** {@inheritDoc} */
     @Override
     public void onModuleLoad() {
+        ScriptInjector.fromString(GwtBootstrap3ClientBundle.INSTANCE.gwtBootstrap3().getText())
```

Comment:
Do we need to check if bootstrap is loaded first here?  I really don't like this type of automatic injection and a bunch of our posted issues have to deal with scripts getting injected multiple times.

Reply:
Hi, this is not THE Bootstrap script, it's [gwtbootstrap3.js](https://github.com/gwtbootstrap3/gwtbootstrap3/pull/518/files#diff-746d31bf5e9da6ea498447efa08d4025) new script that was added in resources.
I didn't knew the best way to add it, so I followed the solution already used in jQuery and Bootstrap scripts.